ansible/configs/sandbox/files/cloud_providers/ec2_cloud_template.j2

```diff
@@ -0,0 +1,402 @@
+#jinja2: lstrip_blocks: "True"
```

Comment:
This seems odd? The sandbox requires a custom ec2_cloud_template.j2?

Reply:
updated